packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -296,6 +296,65 @@ test('Ratio', async t => {
   );
 });
 
+test('Record', async t => {
+  const epRecord = harden({
+    A1: 'Magical Mystery Tour',
+    A2: 'Your Mother Should Know',
+    B: 'I Am the Walrus',
+    C1: 'The Fool on the Hill',
+    C2: 'Flying',
+    D: 'Blue Jay Way',
+  });
+  const paramManager = makeParamManagerBuilder()
+    .addRecord('BestEP', epRecord)
+    .build();
+  // @ts-ignore intentional error
```

Comment:
```suggestion
  // @ts-expect-error
```

Reply:
changing it made the error go away, which is the point of `expect-error`.